Content/posts/error-while-loading-shared-libs.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2099-07-18 09:00
+description: As little as possible, but no less
+tags: TBD
+author: Galen
```

Comment:
Can you add a new field below called `author_username` that has your GitHub username and one for `author_twitter` for Twitter if you want to have those linked. We don't use it yet, but we will soon

Reply:
I'm not active on Twitter, so I'm happy to leave that to the next guest author.

How about `author_github` instead of `author_username`?

Reply:
Yeah that makes more sense